### PR TITLE
add golint in ci

### PR DESCRIPTION
**Pull request by xingao267**
Fixes https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/466

### CODE REVIEW

File: internal/policygen/policygen.go [hunk 1]
```diff
@@ -43,7 +43,6 @@ type RunArgs struct {
 	OutputPath string
 }
 
-// Run executes main policygen logic.
```
Comment (umairidris) [inline internal/policygen/policygen.go:46]: This is a public func. It doesn't need a comment?

File: internal/runner/runner.go [hunk 2]
```diff
@@ -75,15 +75,18 @@ func (d *Default) CmdCombinedOutput(cmd *exec.Cmd) ([]byte, error) {
 // Dry is the Runner that only prints commands and does not execute them.
 type Dry struct{}
 
+// CmdRun prints the command.
```
Comment (umairidris) [inline internal/runner/runner.go:78]: Runs?

File: internal/runner/runner.go [hunk 3]
```diff
@@ -75,15 +75,18 @@ func (d *Default) CmdCombinedOutput(cmd *exec.Cmd) ([]byte, error) {
...
 func (*Dry) CmdRun(cmd *exec.Cmd) error {
 	log.Printf("%v", cmd.String())
 	return nil
 }
 
+// CmdOutput prints the command.
```
Comment (umairidris) [inline internal/runner/runner.go:84]: Runs the command and captures stdout

File: internal/policygen/policygen.go (hunk 1 above)
Comment (xingao267) [reply internal/policygen/policygen.go]: it needs. you caught this right in the meantime of me testing the CI should fail this case.

File: internal/runner/runner.go [hunk 4]
```diff
@@ -75,15 +75,18 @@ func (d *Default) CmdCombinedOutput(cmd *exec.Cmd) ([]byte, error) {
...
 func (d *Dry) CmdOutput(cmd *exec.Cmd) ([]byte, error) {
 	return []byte(cmd.String()), d.CmdRun(cmd)
 }
 
+// CmdCombinedOutput prints the command.
```
Comment (umairidris) [inline internal/runner/runner.go:89]: Runs the command and captures stdout and stderr.

File: internal/policygen/policygen.go (hunk 1 above)
Comment (xingao267) [reply internal/policygen/policygen.go]: It's reverted in the next commit.

File: internal/runner/runner.go (hunk 2 above)
Comment (xingao267) [reply internal/runner/runner.go:78]: It's dry run, so it's just printing the commands.

File: internal/runner/runner.go (hunk 3 above)
Comment (xingao267) [reply internal/runner/runner.go:84]: ditto

File: internal/runner/runner.go (hunk 4 above)
Comment (xingao267) [reply internal/runner/runner.go:89]: ditto

**Comment by xingao267** [comment]
@googlebot rescan

**Pull request merged**